frontend/src/app/modules/games/games-actions.js

```diff
@@ -0,0 +1,8 @@
+import { createActions } from 'reduxsauce'
+
+export const GamesActions = createActions({
+  gamesLoaded: ['games'],
+  selectGame: ['selectedGame'],
+}, {
+  prefix: 'foosball-rating/games/',
```

Comment:
small issue - I might consider removing the "foosball-rating" prefix, as we're considering to rename the application and you don't want to leave traces like that ;)

Reply:
wouldn't it be hard to find where you need to rename/add prefix unless you have each place with an old name?